ynr/apps/uk_results/management/commands/import_2010_parl_results.py

```diff
@@ -92,7 +92,7 @@ def create_results(self, ballot, candidates):
 
             LoggedAction.objects.create(
                 user=resultset.user,
-                action_type="entered-results-data",
+                action_type=ActionType.ENTERED_RESULTS_DATA,
```

Comment:
After much experimentation, I *think* this is the way to reference labels. 

Note to Monday me: Go through local uncommitted. changes and follow this pattern.